drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/mci_payload.rb

```diff
@@ -18,8 +18,8 @@ def self.from_client(client, mci_id: nil)
         'middleName' => client.middle_name,
         'lastName' => client.last_name,
         'suffix' => client.name_suffix,
-        'ssn' => client.ssn,
-        # 'ssnAlias' => client.ssn,
+        # API only accepts full SSNs
+        'ssn' => client.ssn&.match?(/^\d{9}$/) ? client.ssn : nil,
```

Comment:
If the ssn is potentially dirty, perhaps add a method on client to strip out hyphens, spaces, etc.

```
def normalized_ssn
  result = ssn&.gsub(/[^0-9]/, '')
  result && result.size == 9 ? result : nil
end
```

Reply:
It should already be clean at this point, HMIS specs allows partial ssns though by replacing missing numbers with "x" or "X"